packages/smooth_app/lib/pages/folksonomy/folksonomy_card.dart

```diff
@@ -0,0 +1,143 @@
+        }
+
+        Navigator.of(context)
+            .push(
+              MaterialPageRoute<void>(
+                builder: (BuildContext context) => FolksonomyPage(product),
+              ),
+            )
+            .then((void completion) => <Set<Future<void>>>{
+                  if (context.mounted)
+                    <Future<void>>{
+                      Provider.of<FolksonomyProvider>(context, listen: false)
+                          .fetchProductTags(),
+                    }
+                });
+      },
+      child: Padding(
+        padding: const EdgeInsets.only(top: LARGE_SPACE),
+        child: buildProductSmoothCard(
+          title: Stack(
+            children: <Widget>[
+              Center(child: Text(appLocalizations.product_tags_title)),
+            ],
+          ),
+          body: Container(
+            width: double.infinity,
+            padding: const EdgeInsetsDirectional.all(LARGE_SPACE),
+            child: Column(
+              crossAxisAlignment: CrossAxisAlignment.start,
+              mainAxisAlignment: MainAxisAlignment.start,
+              children: <Widget>[
+                CardList(),
+              ],
+            ),
+          ),
+        ),
+      ),
+    );
+  }
+}
+
+class CardList extends StatelessWidget {
+  @override
+  Widget build(BuildContext context) {
+    final AppLocalizations appLocalizations = AppLocalizations.of(context);
+    final FolksonomyProvider provider = context.watch<FolksonomyProvider>();
+
+    if (provider.isLoading) {
+      return const Center(child: CircularProgressIndicator.adaptive());
+    } else if (!provider.tagsExist) {
+      return Center(
+        child: Row(
+          mainAxisAlignment: MainAxisAlignment.center,
+          children: <Widget>[
+            Expanded(
+              child: Text(
+                appLocalizations.no_product_tags_found_message,
+                textAlign: TextAlign.center,
+              ),
+            ),
+            Icon(ConstantIcons.instance.getForwardIcon()),
+          ],
+        ),
+      );
+    } else {
+      final Map<String, ProductTag> tags = provider.productTags!;
+      final Iterable<MapEntry<String, ProductTag>> displayTags =
+          tags.entries.toList().take(5);
+
+      return Padding(
+        padding: const EdgeInsetsDirectional.only(
+          top: VERY_SMALL_SPACE,
+          bottom: VERY_SMALL_SPACE,
+        ),
+        child: Semantics(
+          button: true,
+          container: true,
+          excludeSemantics: true,
+          child: Row(
+            children: <Widget>[
+              Expanded(
+                child: Padding(
+                  padding: const EdgeInsets.all(SMALL_SPACE),
+                  child: Column(
+                    crossAxisAlignment: CrossAxisAlignment.start,
+                    children:
+                        displayTags.map((MapEntry<String, ProductTag> entry) {
+                      return Tag(text: '${entry.key}: ${entry.value.value}');
```

Comment:
For the pattern, please translate it.
Eg: in French, we say '${entry.key} : ${entry.value.value}'

Reply:
Not sure I understand. These are values that exist in the backend and could be anything, where adding something in app_en.arg wouldnt make sense. Or were you thinking of translating these values during runtime?

Reply:
You have hardcoded the following pattern: "XX: XX".
But for example, in French, it should be "XX : XX" (a space before and after)
Hence, the need to translate the pattern.

Reply:
For that you can reuse `sep`:
```
    "sep": "",
    "@sep": {
        "description": "Separator just before a colon (':'). Probably only populated in French and empty in other languages."
    },
```

Reply:
so 

`'${entry.key}${appLocalizations.sep}: ${entry.value.value}');`

?

Reply:
cool updated